tests/unit/linalg/test_matrix_rectangular.cpp

```diff
@@ -102,7 +97,7 @@ TEST_CASE("ParallelFormRectangular",
    {
       Mesh mesh(10, 10, Element::QUADRILATERAL, 0, 1.0, 1.0);
       int dim = mesh.Dimension();
-      int order = 2;
+      int order = 4;
```

Comment:
Out of curiosity why bump the order up here?

Reply:
The previous tests did not use polynomials, so the results were not exact. This wasn't evident because of the loose default tolerance in the previous version of Catch. I changed the examples to use polynomials so the answer can be reproduced exactly, but only if the FE space has sufficiently high order.